examples/storage/cassandra/README.md

```diff
@@ -380,10 +380,10 @@ spec:
   replicas: 4
 ```
 
-The Pet Set will now contain four Pets.
+The StatefulSet will now contain four pods.
```

Comment:
These are not pods.  What are we calling them?  We have replicas in a deployment.  But each replica contains a pod, also you can have multiple pods in a single Pet.